### DISCUSSION TITLE
Give dividers a more meaningful interface on the data model config page

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#16171] [Feature Requests]
Pretty simple request. I have a `products` collection whose detail page is already getting pretty bloated with fields organized under multiple sections, where each "section" is denoted by a `divider` alias field.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/26592486/197885624-70ce6943-ed52-4dab-a44c-89064ccd26f5.png">

As you can see, in the "data model view" the divider field has an un-configurable ID like `divider-{letters}` that doesn't tell me anything about what the divider text actually says. For this particular divider I've set a **Title** of "Publishing", but right now I have to just infer that information from my knowledge of the fields surrounding it. It would be much easier on my brain if:

- The UI showed the **Title** of the divider somewhere in addition to or instead of the ID.
- The ID was configurable so I could make it more meaningful.

**Comment by connorwinston** [comment]
I agree with the UX of this not being the best, which is a side affect of the new field layout which auto-gens the ID. However, you can set the ID of your self if you create a `Presentation Alias Field` using the Advanced Field Creator.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/12714889/197900291-e90d9dca-c9a5-4511-a6e5-8255a518fda7.png">

![image](https://user-images.githubusercontent.com/12714889/197900338-9d73eeae-6be7-4618-b0e0-2efcda373811.png)

**Reply by AaronBeaudoin** [reply]
This is a decent workaround, but I maintain that since "groups" have a special representation on the data model config page, it is reasonable to expect dividers to also have a special representation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
